products/mozilla_org/test_mozilla_org.md

```diff
@@ -0,0 +1,67 @@
+# Testing Mozilla.org
+
+Mozilla.org is highly visible because the site houses the basic info of all Mozilla produced products, conveys Mozilla’s mission, vision and values it stands for. Additionally, it promotes initiatives and campaigns in time of these events. As a result, new pages are created. New content is added to existing pages. New layout is introduced. The localized versions reach 60% of the Mozilla users globally. It is very important that, not only the main pages are localized, they are thoroughly tested before they are launched on production.  
+
+## Key Links
+* Production: https://www.mozilla.org/(locale)/
+* Staging: https://www-dev.allizom.org/(locale)/
+* Repository:  https://github.com/mozilla-l10n/www.mozilla.org/{local code}/{project name}/{sub pages}
+* Pontoon: https://pontoon.mozilla.org/(local code)/mozillaorg/{project name}/{sub pages}
```

Comment:
I would remove the (locale code) from all other URLs, you can point them to https://pontoon.mozilla.org/projects/mozillaorg/ for Pontoon